aws-codeartifact-domain/src/main/java/software/amazon/codeartifact/domain/Translator.java

```diff
@@ -58,12 +58,9 @@ static CreateDomainRequest translateToCreateRequest(final ResourceModel model) {
   /**
    * Request to read a resource
    * @param model resource model
-   * @param request resource handler request
    * @return awsRequest the aws service request to describe a resource
    */
-  static DescribeDomainRequest translateToReadRequest(
-      final ResourceModel model, final ResourceHandlerRequest<ResourceModel> request
```

Comment:
`request` was not being used, doesn't really have to do with the description of the PR